src/TouristInfoCards.js

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+
+const TouristInfoCards = () => {
+  const cities = [
+    {
+      id: 1,
+      name: "Glasgow",
+      img:
+        "https://www.visitbritain.com/sites/default/files/consumer/paragraphs-bundles/image-header-with-text/glasgow_clyde_arc_vb34161123.jpg",
+      info:
+        "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus ut velit pharetra, blandit erat",
+      link: "https://peoplemakeglasgow.com/"
+    },
+    {
+      id: 2,
+      name: "Manchester",
+      img:
+        "https://eu-assets.simpleview-europe.com/manchester2016/imageresizer/?image=%2Fdbimgs%2FManOne-03%284%29.jpg&action=Background_Overlay",
+      info:
+        "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus ut velit pharetra, blandit erat",
+      link: "https://www.visitmanchester.com/"
+    },
+    {
+      id: 3,
+      name: "London",
+      img:
+        "https://cdn.londonandpartners.com/-/media/images/london/visit/things-to-do/sightseeing/london-attractions/tower-bridge/tower-bridge-shutterstock-1920x582.jpg?mw=1920&hash=078DDB24AC457E2D8F231FC3E0F7EC55A00C88E3",
+      info:
+        "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Phasellus ut velit pharetra, blandit erat",
+      link: "https://www.visitlondon.com/"
+    }
+  ];
+  return (
```

Comment:
Looks good, I would be inclined to make the cards componants in their own right, but this works fine